apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -185,31 +102,39 @@ function transformToPlainObject (input) {
     return result
 }
 
+const isDateStrValid = (dateStr) => {
+    return isDateStrValidUtils(dateStr, { utc: true, formats: [DATE_FORMAT] })
```

Comment:
Why iso is not passed

Reply:
If you are talking about YYYY-MM-DDTHH:mm:ss.SSSZ format in formats array, because parsing logic for utc format is different and passing this format to dayjs in strict mode has no use